code/game/gamemodes/nations/nations.dm

```diff
@@ -0,0 +1,164 @@
+datum/game_mode/nations
+	name = "nations"
+	config_tag = "nations"
+	required_players_secret = 25
+	var/const/waittime_l = 1200 //lower bound on time before intercept arrives (in tenths of seconds)
```

Comment:
The waittimes vary from gamemode to gamemode. I guess they could be standardized, but that's outside the scope of this PR.